bazel/repository_locations.bzl

```diff
@@ -127,10 +127,10 @@ REPOSITORY_LOCATIONS = dict(
         urls = ["https://github.com/h2o/picohttpparser/archive/1d2b8a184e7ebe6651c30dcede37ba1d89691351.tar.gz"],
     ),
     com_github_iovisor_bcc = dict(
-        sha256 = "d34f9484588a9c25be936c910c86f8b25b04e5b0c802d0630e77cc9a8a272aed",
-        strip_prefix = "bcc-e0698be7b797129cb113912e96ad741a551e2291",
+        sha256 = "416426fbe22d617d8aed088062f4489e69176136e99dc0b933df58e83d9175da",
+        strip_prefix = "bcc-e57be8465b9cf238f1c04b1c7e154fd1db85326d",
         urls = [
-            "https://github.com/pixie-io/bcc/archive/e0698be7b797129cb113912e96ad741a551e2291.tar.gz",
+            "https://github.com/pixie-io/bcc/archive/e57be8465b9cf238f1c04b1c7e154fd1db85326d.tar.gz",
         ],
     ),
```

Comment:
This is the [pixie9](https://github.com/pixie-io/bcc/tree/pixie9) branch, which rebased our patches on top of v0.31.0 tag (latest bcc release).